shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/model/PaginationMetaApiModel.kt

```diff
@@ -0,0 +1,18 @@
+package co.nimblehq.blisskmmic.data.model
+
+import co.nimblehq.blisskmmic.domain.model.PaginationMeta
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+class PaginationMetaApiModel(
+    val page: Int,
+    val pages: Int,
+    @SerialName("page_size")
+    val pageSize: Int,
+    val records: Int
+)
```

Comment:
This single annotation `@SerialName` looks a bit odd in the overall class, maybe we should serialize all for consistency even though it might duplicate a bit:
```suggestion
class PaginationMetaApiModel(
   @SerialName("page") val page: Int,
    @SerialName("pages") val pages: Int,
    @SerialName("page_size") val pageSize: Int,
    @SerialName("records") val records: Int
)
```
What do you think? 🤔

Reply:
I really don't like this, it adds too much to read, what does not have `SerialName` is default-ly define by the val name. `SerialName` is added when needed, is the way I see this. 🤔

Reply:
Noted, then how about making the line that has `SerialName` into one line to show the relationship between the attribute and the `SerialName` annotation a bit better? I am okay with your reasons for skipping the attributes that don't need the `SerialName` 👍
```suggestion
class PaginationMetaApiModel(
    val page: Int,
    val pages: Int,
    @SerialName("page_size") val pageSize: Int,
    val records: Int
)
```